client/app/utils/metaTagsManager.js

```diff
@@ -1,3 +1,5 @@
+import _find from 'lodash/collection/find';
```

Comment:
According to the [lodash docs](https://lodash.com/#installation), loading only the required method is "great for smaller builds with browserify/webpack". I kept the `_` in the method name just for consistency with the library.